netbox_cmdb/netbox_cmdb/template_content.py

```diff
@@ -1,15 +1,22 @@
 from extras.plugins import PluginTemplateExtension
 
 
-class Decommisioning(PluginTemplateExtension):
-    model = "dcim.device"
-
+class DecommisioningBase(PluginTemplateExtension):
     def buttons(self):
         return (
-            f'<a href="#" hx-get="/plugins/cmdb/decommisioning/{self.context["object"].id}/delete" '
-            'hx-target="#htmx-modal-content" class="btn btn-sm btn-danger" data-bs-toggle="modal" data-bs-target="#htmx-modal" '
+            f'<a href="/plugins/cmdb/decommisioning/{self.obj}/{self.context["object"].id}/delete" '
```

Comment:
i believe there are 2 "s" in decommissioning (same comment applies to all class containing Decommission in their names)

Reply:
yep, typo + copy pasta from here ^^
but at least the filenames are correct xD

Reply:
done